config/sync/field.field.user.user.field_u_policy_agreement.yml

```diff
@@ -0,0 +1,24 @@
+uuid: 7aef2d34-f8b7-477b-9811-05c872ed7754
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.user.field_u_policy_agreement
+  module:
+    - user
+id: user.user.field_u_policy_agreement
+field_name: field_u_policy_agreement
+entity_type: user
+bundle: user
+label: 'Policy Agreement'
+description: 'I have read and agree to the <a href="/terms-and-conditions">terms</a> and <a href="/privacy-policy">privacy policy</a> of Easy Recipe Delight.'
```

Comment:
Find out if there's a way to add the description above the actual element, because it makes more sense to have it before or change the actual label of the element to be what your description is.

Reply:
Is this okay?